contrib/README.md

```diff
@@ -0,0 +1,36 @@
+This folder contains contributed files for use with Sopel
```

Comment:
```suggestion
This folder contains contributed files for use with Sopel.
```

A top-level header could be nice, too, since you've pushed into Markdown territory anyway.

But maybe you didn't add a `# Header` to the top of the file because, like me, you couldn't think of a good heading to put there? 😅

Eh, this isn't a blocker, just wondering if you thought of it.

Reply:
Yea, I can't think of a meaningful one to put there :stuck_out_tongue: